src/main/java/com/icthh/xm/ms/entity/web/rest/LinkResource.java

```diff
@@ -65,6 +68,7 @@ public ResponseEntity<Link> createLink(@Valid @RequestBody Link link) throws URI
                                         "A new link cannot already have an ID");
         }
         Link result = linkService.save(link);
+        Hibernate.initialize(result.getTarget());
```

Comment:
just @Transactional not solve the problem?

Reply:
doesn't in runtime